code/obj/item/staffkiosk.dm

```diff
@@ -0,0 +1,287 @@
+		/obj/item/device/radio/headset/medical,
+		/obj/item/disk/data/cartridge/medical,
+		/obj/item/reagent_containers/emergency_injector/epinephrine,
+		/obj/item/reagent_containers/mender/brute,
+		/obj/item/reagent_containers/mender/burn,
+		/obj/item/bandage)
+
+/obj/item/storage/box/staffkit/robo
+	name = "robotics assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffmed,
+		/obj/item/device/radio/headset/medical,
+		/obj/item/disk/data/cartridge/medical,
+		/obj/item/screwdriver,
+		/obj/item/cable_coil,
+		/obj/item/body_bag)
+
+/obj/item/storage/box/staffkit/eng
+	name = "engineering assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffeng,
+		/obj/item/device/radio/headset/engineer,
+		/obj/item/disk/data/cartridge/engineer,
+		/obj/item/clothing/head/helmet/hardhat,
+		/obj/item/crowbar,
+		/obj/item/screwdriver,
+		/obj/item/device/multitool)
+
+/obj/item/storage/box/staffkit/mech
+	name = "mechanics assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffeng,
+		/obj/item/device/radio/headset/engineer,
+		/obj/item/disk/data/cartridge/mechanic,
+		/obj/item/electronics/soldering,
+		/obj/item/device/t_scanner,
+		/obj/item/screwdriver,
+		/obj/item/device/multitool)
+
+/obj/item/storage/box/staffkit/miner
+	name = "mining assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffeng,
+		/obj/item/device/radio/headset/engineer,
+		/obj/item/clothing/head/emerg,
+		/obj/item/clothing/suit/space/emerg,
+		/obj/item/crowbar,
+		/obj/item/mining_tool/power_pick,
+		/obj/item/satchel/mining)
+
+/obj/item/storage/box/staffkit/res
+	name = "research assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffres,
+		/obj/item/device/radio/headset/research,
+		/obj/item/clothing/mask/gas,
+		/obj/item/clothing/gloves/latex,
+		/obj/item/reagent_containers/glass/beaker,
+		/obj/item/device/reagentscanner)
+
+/obj/item/storage/box/staffkit/chef
+	name = "culinary assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffciv,
+		/obj/item/device/radio/headset/civilian,
+		/obj/item/clothing/head/souschefhat,
+		/obj/item/clothing/gloves/latex,
+		/obj/item/satchel/hydro,
+		/obj/item/reagent_containers/food/drinks/drinkingglass/icing,
+		/obj/item/spraybottle/cleaner)
+
+/obj/item/storage/box/staffkit/bar
+	name = "bar assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffciv,
+		/obj/item/device/radio/headset/civilian,
+		/obj/item/clothing/suit/wcoat,
+		/obj/item/clothing/gloves/black,
+		/obj/item/device/reagentscanner,
```

Comment:
`/obj/item/paper/book/pocketguide/bartending		`
`/obj/item/device/reagentscanner` Not sure if this is really necessary. 
`/obj/item/clothing/gloves/black` Why free gloves?
`/obj/item/spraybottle/cleaner` Well I guess, this gives 'em something to do...	 Sponge feels a bit better.

Reply:
Bartenders require reagent scanners because food and drink are frequently poisoned and there is only one available.
The cleaner bottle is a thematic component and so are the gloves.

Reply:
The gloves are for table serving purposes, but are taking up a slot that I think would be better used for the bartending guide, so I'll swap that. As to the other stuff: reagent scanner is pretty necessary, and while the idea of cleaning up vomit with a sponge is gut-wrenchingly amusing, I think for this particular task a cleaner bottle is a bit more suitable.